packages/docs-site/src/components/presenters/sketch-widget/index.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import './sketch-widget.scss'
+
+import SketchLogo from './sketch-logo.svg'
+import DownloadIcon from './download-icon.svg'
+
+const SketchWidget = ({ name, href }) => {
+  return (
+    <article className="sketch-widget">
```

Comment:
I left the rn- part of the classname in footer. wasn't sure if we wanted to namespace or not. doesn't matter

Reply:
I wanted to keep a distinction between the namespaced component-library styles and those within this codebase.